rbc/src/main/java/com/library/rbc/model/Author.java

```diff
@@ -0,0 +1,18 @@
+package com.library.rbc.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Author {
+
+    @Id
+    private String id;
+    private String fullName;
```

Comment:
Suggestion: split `fullName` into two fields, `firstName` and `lastName.`

Reply:
Author is always added through one input field (full name) on UI, so full name could be simpler and more practical.

Reply:
Sounds good.